### DISCUSSION TITLE
Mesh Central don`t start after config.json change

### DISCUSSION THREAD

**Discussion by wireks** [Ylianst/MeshCentral#5753] [General]
Mesh Central don`t start after config.json change. How to troubleshout ?

**Comment by si458** [comment]
use something like https://jsoneditoronline.org/ to edit your json
and then something like https://jsonlint.com/ to validate it

**Comment by simpsonetti** [comment]
I regular work with vim for editing files. 
But i have also VS code and using this for the config.json is awesome:
https://ylianst.github.io/MeshCentral/meshcentral/tipsntricks/#fancy-config-editing-with-vs-code
Hope it helps.
